gym/wrappers/common/frame_skipping.py

```diff
@@ -0,0 +1,79 @@
+import gym
+
+__all__ = ['Skip', 'Skip1', 'Skip2', 'Skip3', 'Skip4', 'Skip5', 'Skip6', 'Skip7', 'Skip8', 'Skip9', 'Skip10']
+
+class Skip(gym.Wrapper):
+    """
+        Generic common frame skipping wrapper
+        Will perform action for `x` additional steps
+    """
+    def __init__(self, env, repeat_for_x_steps):
+        super(Skip, self).__init__(env)
+        self.repeat_count = repeat_for_x_steps
+        self.record_stepcount = not hasattr(env.unwrapped, 'stepcount')
+        if self.record_stepcount:
+            env.unwrapped.stepcount = 0
```

Comment:
To avoid the issue of incorrect stepcount if they are stacked. But that's not really an issue. I'll fix.